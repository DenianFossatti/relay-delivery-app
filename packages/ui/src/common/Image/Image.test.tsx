import { mockTestID, render, wrapper } from '../../tests';

import { Image, ImageProps } from '.';

describe(`Image`, () => {
  const props: ImageProps = {
    alt: 'Image',
    source: { uri: 'https://via.placeholder.com/300x200' },
    testID: 'Image',
  };

  const { findByTestId, toJSON } = render(<Image {...props} />, { wrapper });

  const el_container = mockTestID('Image', props.testID!);

  it(`render component #${el_container}`, async () => {
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
  });

  it('should render snapshot', () => {
    expect(toJSON()).toMatchSnapshot();
  });
});
