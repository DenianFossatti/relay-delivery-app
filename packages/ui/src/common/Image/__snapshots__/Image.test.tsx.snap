// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Image should render snapshot 1`] = `null`;
